projects/baklava/_posts/2020-10-05-c.md

```diff
@@ -2,13 +2,13 @@
 title: Baklava in C
 layout: default
 date: 2020-10-05
-featured-image:
+featured-image: baklava-in-every-language.jpg
```

Comment:
I think this is fine! In fact, this is much better than the current situation. That said, ideally we'd want new images for this articles.

Reply:
Yes, we do not have featured images for some articles. There the image titled 'Sample Programs in Every Language' is displayed.